search/lib/api/provider.js

```diff
@@ -53,7 +55,14 @@ async function getProvider (request, response) {
       title: providerId,
       description: `Root catalog for ${providerId}`,
       stac_version: settings.stac.version,
-      links: [...links, ...childLinks]
+      links: [...links, ...childLinks],
+      conformsTo: [
+        'https://api.stacspec.org/v1.0.0-beta.1/core',
```

Comment:
https://api.stacspec.org/v1.0.0-beta.1/core
https://api.stacspec.org/v1.0.0-beta.1/item-search
https://github.com/radiantearth/stac-api-spec
Is listed in the ticket, I'm guessing this is the fully updated list of links?

Reply:
The first 2 are the conformance classes for STAC, while the others are if you implement all the endpoints for an OGC Features API which we have here.